backend/coreapp/views.py

```diff
@@ -337,3 +343,29 @@ def user(request, username):
     """
 
     return Response(serialize_profile(request, get_object_or_404(Profile, user__username=username)))
+
+def export(request, slug):
```

Comment:
Can you throw the last_modified/etag stuff that scratch api uses on this endpoint?

Reply:
This endpoint should also be limited to get requests

Reply:
Refactored, and moved `scratch_condition` outside of the `ScratchDetail` class to share it?

Reply:
ty